Office 365/_meta/manifest.yml

```diff
@@ -1,4 +1,9 @@
 uuid: cfbfe5d6-64f1-41f2-ba04-38909bcae5e7
 name: Office 365
 slug: o365
-description: ''
+description: >-
+  Microsoft 365 is an online service, providing the Microsoft Office Products.
+
+  Providing Microsoft 365 activity logs is very valuable source of information for SEKOIA.IO.
+  It will help detection of account breaches (e.g. impossible travel activity),
+  phishing attempts, data theft. 
```

Comment:
```suggestion
  Microsoft 365 is an online service, providing the Microsoft Office Products.

  Microsoft 365 activity logs are a very valuable source of information for SEKOIA.IO.
  It could for example help in the detection of account breaches (e.g. impossible travel activity),
  phishing attempts and data theft. 
```

Reply:
Second paragraph removed (Not relevant)